go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/zclconf/go-cty v1.14.4
 	go.mongodb.org/atlas v0.36.0
 	go.mongodb.org/atlas-sdk/v20231115014 v20231115014.0.0
+	go.mongodb.org/atlas-sdk/v20240530001 v20240530001.0.1-0.20240619083352-d423f3e4cbe1
```

Comment:
nit: do we plan to have a task to undo this and upgrade?

Reply:
Are you referring to eventually removing the preview and adopting the new stable sdk release when ISS is released? Can create a ticket for this.